.github/workflows/test_production_function.yaml

```diff
@@ -6,206 +6,172 @@
 
 name: Functional Tests
 
-# Run this workflow at 12:15 am
-# on every Sun to Thr (b/c GMT -5)
 on:
+  # Run this workflow at 12:15 am
+  # on every Sun to Thr (b/c GMT -5)
   schedule:
     - cron: "15 4 * * 0-4"
+  # This allows someone to run tests manual and enter their own email,
```

Comment:
This comment is a bit misleading, as currently I can't enter my email to get the failure emails, I can only turn on or off sending the emails to TCOs